setup.py

```diff
@@ -6,16 +6,23 @@
     packages=['mycroft_voice_satellite', 'mycroft_voice_satellite.speech',
               'mycroft_voice_satellite.res',
               'mycroft_voice_satellite.configuration'],
-    install_requires=["jarbas_hive_mind>=0.10.3",
-                      "speech2text",
+    install_requires=["jarbas_hive_mind",
+                      "SpeechRecognition @ git+https://github.com/Uberi/speech_recognition.git",
                       "pyee",
                       "requests",
                       "requests_futures",
                       "psutil",
-                      "PyAudio==0.2.11",
-                      "pocketsphinx==0.1.15",
+                      "PyAudio",
+                      "pocketsphinx",
+                      "json_database",
+                      "jarbas_utils",
```

Comment:
jarbas_utils has been replaced by ovos_utils

Reply:
Replaced it in the code so only ovos_utils is used